### PR TITLE
lsd2dsl: 0.5.2 → 0.5.4

### PR DESCRIPTION
**Pull request by sikmir**
###### Motivation for this change
[Changelog](https://github.com/nongeneric/lsd2dsl/releases)

###### Things done
- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [x] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 134364` at c1f7d027 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built successfully:</summary>
  <ul>
    <li>lsd2dsl</li>
  </ul>
</details>
<details>
  <summary>1 suggestion:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/missing-phase-hooks.md">missing-phase-hooks</a>

`installPhase` should probably contain `runHook preInstall` and `runHook postInstall`.

Near pkgs/applications/misc/lsd2dsl/default.nix:30:3:
```
   |
30 |   installPhase = ''
   |   ^
```
</li>
  </ul>
</details>

File: pkgs/applications/misc/lsd2dsl/default.nix [hunk 1]
```diff
@@ -33,6 +44,6 @@ mkDerivation rec {
     '';
     license = licenses.mit;
     maintainers = with maintainers; [ sikmir ];
-    platforms = with platforms; linux ++ darwin;
+    platforms = with platforms; unix;
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/lsd2dsl/default.nix:47]:
```suggestion
    platforms = platforms.unix;
```

File: pkgs/applications/misc/lsd2dsl/default.nix [hunk 2]
```diff
@@ -18,12 +18,23 @@ mkDerivation rec {
...
+
   installPhase = ''
     install -Dm755 console/lsd2dsl gui/lsd2dsl-qtgui -t $out/bin
-  '' + lib.optionalString stdenv.isDarwin ''
+  '' + (if stdenv.isDarwin then ''
     wrapQtApp $out/bin/lsd2dsl
     wrapQtApp $out/bin/lsd2dsl-qtgui
-  '';
+  '' else ''
+    install -Dm644 {${desktopItem},$out}/share/applications/lsd2dsl.desktop
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/lsd2dsl/default.nix:36]: Use copyDesktopItems instead.

Comment (sikmir) [reply pkgs/applications/misc/lsd2dsl/default.nix:36]: Thanks, didn't know about copyDesktopItems.

**Pull request merged**